packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -145,10 +145,69 @@ const signSession = (
     account.rawLiquidityClue = payload.rawLiquidityClue;
     account.session = {
         ...account.session,
-        signedRounds: account.session.signedRounds.concat(payload.roundId),
+        signedRounds: account.session.signedRounds.concat({
```

Comment:
note: this is breaking change for storage, do we care about migration at this point?